src/backend/opbeat_backend.js

```diff
@@ -153,7 +153,7 @@ function groupingTs (ts) {
 
 function transactionGroupingKey (trans) {
   return [
-    groupingTs(trans._startStamp).getTime(),
+    groupingTs(trans.transaction._startStamp).getTime(),
```

Comment:
This should be `trans._startStamp`, I can fix and amend this commit but then it would overwrite me as the author.
Do you want to do it yourself?